web/satellite/src/api/referral.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import apollo from '@/utils/apolloManager';
+import gql from 'graphql-tag';
+import { RequestResponse } from '@/types/response';
+
+export async function fetchReferralInfo(): Promise<RequestResponse<ReferralInfo>> {
+    let result: RequestResponse<ReferralInfo> = new RequestResponse<ReferralInfo>();
+
+    let response: any = await apollo.query(
+        {
+            query: gql(`
+                query {
+                    activeReward (
+                        type: 1,
```

Comment:
Can we make this type info to be dynamic and then create a constant value to hold the type info? That way, we can use this query to query activeReward for other types too